doc/submit-a-report.rst

```diff
@@ -0,0 +1,100 @@
+###############
+Submit a Report
+###############
+
+If you can't fix your issue with any of the troubleshooting steps, then you'll want to submit it to our issue tracker so our team can take a look at it for you.
+You'll need a `Github account`_ to do this, so make sure you have one before you begin (a good idea anyway).
+
+.. image:: images/submit-a-report/submit-a-report-issue-tracker.png
+   :alt: Github page on Spyder's issue tracker
+
+.. important::
+
+   Before you submit an issue, make sure you've searched a description of the problem and a relevant portion of the error traceback, on both Google and Spyder's `issue tracker`_ to make sure it hasn't been submitted before.
+   If that's the case, your issue will be closed as a duplicate.
+
+.. _Github account: https://github.com/
+.. _issue tracker: https://github.com/spyder-ide/spyder/issues
+
+
+
+=======================
+Ways to submit an issue
+=======================
+
+There are several ways to submit an issue, either from Spyder or GitHub directly.
+In order of preference and difficulty:
+
+#. **If Spyder presents an error dialog** you can submit an issue directly from it.
+   You will have to fill out a title for your issue, specify the steps that lead to this problem and click :guilabel:`submit to Github`.
+   This will prefill an error report with your environment details, key versions and dependencies and automatically insert the error/traceback for you.
+
+   .. image:: images/submit-a-report/submit-a-report-error-dialog.png
+      :alt: Github page on Spyder's issue tracker
+      :width: 500px
+
+#. **If Spyder opens and your issue does not involve an error dialog**, the best way to do so is to simply select :guilabel:`Report issue` from the :guilabel:`Help` menu, which manually brings up the issue report form and fills in the key information about your Spyder installation.
+   Describe the issue you're experiencing (including any error/traceback information) along with a descriptive title, and click :guilabel:`Submit to Github`.
+
+   .. image:: images/submit-a-report/submit-a-report-issue.png
+      :alt: Github page on Spyder's issue tracker
+
+#. **If Spyder won't launch**, you can submit a report manually at our issues page on `Github`_.
+   Unlike the above, you'll need to manually provide the versions of everything (Spyder, Python, OS, Qt/PyQt, Anaconda, and Spyder's dependencies) as listed in the error report template; see below for more on that.
+
+   .. image:: images/submit-a-report/submit-a-report-issue-template.png
+      :alt: Github page on Spyder's issue template
+
+.. _GitHub: https://github.com/spyder-ide/spyder/issues
+
+Once you submit your report, our team will try to get back to you as soon as possible, often within 24 hours or less, to try to help you fix it.
+
+
+What to include in your report
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
==============================
What to include in your report
==============================
```

I think this should an `h2` section instead of `h3`, to look better in the outline:

![Selección_030](https://user-images.githubusercontent.com/365293/90325963-ca6c7900-df47-11ea-82eb-947c543f49ca.png)

Reply:
Agreed